<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientDialog</class>
 <widget class="QDialog" name="PatientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>498</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Добавить/Редактировать пациента</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="secNameLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="noteTextEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>331</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="secNameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Фамилия</string>
   </property>
  </widget>
  <widget class="QLabel" name="firstNameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstNameLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="patronymicLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Отчество</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="patronymicLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="birthYearSpinBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1900</number>
   </property>
   <property name="maximum">
    <number>2100</number>
   </property>
  </widget>
  <widget class="QLabel" name="birthYearLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Год рождения</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="heightSpinBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
  </widget>
  <widget class="QLabel" name="heightLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Рост</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="weightSpinBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
  </widget>
  <widget class="QLabel" name="weightLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Вес</string>
   </property>
  </widget>
  <widget class="QLabel" name="noteLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Заметки</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PatientDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PatientDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
